packages/typeorm/src/types.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// Re-export everything from 'typeorm'
+export * from 'typeorm';
+export {
+  SqliteConnectionOptions,
+  MongoConnectionOptions,
+  MysqlConnectionCredentialsOptions,
+  MysqlConnectionOptions,
+  MysqlQueryRunner,
+  MongoQueryRunner,
+  NativescriptConnectionOptions,
+  NativescriptQueryRunner,
+  MongoDriver,
+  MysqlDriver,
+  NativescriptDriver,
+  OracleConnectionCredentialsOptions,
+  OracleConnectionOptions,
+  OracleDriver,
+  OracleQueryRunner,
+  SqliteDriver,
+  SqliteQueryRunner,
+  PostgresConnectionCredentialsOptions,
+  PostgresConnectionOptions,
+  PostgresDriver,
+  PostgresQueryRunner,
+  ReactNativeConnectionOptions,
+  ReactNativeDriver,
+  ReactNativeQueryRunner,
+  SapConnectionCredentialsOptions,
+  SapConnectionOptions,
+  SapDriver,
+  SapQueryRunner,
+  AbstractSqliteDriver,
+  AbstractSqliteQueryRunner,
+  SqljsConnectionOptions,
+  SqljsDriver,
+  SqljsQueryRunner,
+  MssqlParameter,
+  SqlServerConnectionCredentialsOptions,
+  SqlServerConnectionOptions,
+  SqlServerDriver,
+  SqlServerQueryRunner,
+  AuroraDataApiConnection,
+  AuroraDataApiConnectionCredentialsOptions,
+  AuroraDataApiConnectionOptions,
+  AuroraDataApiDriver,
+  AuroraDataApiQueryRunner,
+  AuroraDataApiPostgresConnectionOptions,
+  AuroraDataApiPostgresQueryRunner,
+  CockroachConnectionCredentialsOptions,
+  CockroachConnectionOptions,
+  CockroachDriver,
+  CockroachQueryRunner,
+  CordovaConnectionOptions,
+  CordovaDriver,
+  CordovaQueryRunner,
+  ExpoConnectionOptions,
+  ExpoDriver,
+  ExpoQueryRunner,
+};
```

Comment:
Do we really need that long list of exports 👆 and the long list of imports 👇 ? Isn't it enough to `export * from 'typeorm';`?

Reply:
Fair enough 🤷